rdr_service/offline/ce_health_data_reconciliation_pipeline.py

```diff
@@ -102,7 +102,7 @@ def _find_latest_24_hours_reconciliation_files(self, prefix):
             raise RuntimeError("No reconciliation files in cloud bucket %r (all files: %s)." % (self.bucket_name,
                                                                                                 bucket_stat_list))
         last_24_hours_files = [blob for blob in bucket_stat_list
-                               if blob.updated >= (self.job_started_time - _MAX_INPUT_AGE)]
+                               if blob.updated.replace(tzinfo=None) >= self.job_started_time - _MAX_INPUT_AGE]
```

Comment:
This should use `app_util.datetime_as_naive_utc` to set the updated datetime to a naive datetime (just in case the updated date has any timezone other than UTC).

Reply:
Good catch, done!